mobile/src/PushNotifications.js

```diff
@@ -290,6 +297,62 @@ class PushNotifications extends Component {
     }
   }
 
+  /**
+   * Generates and returns an auth token
+   */
+  async getAuthToken() {
+    const { wallet } = this.props
+
+    // No active account, can't proceed
+    if (!wallet || !wallet.activeAccount) {
+      console.debug('Cannot generate auth token, no active account')
+      return
+    }
+
+    // No private key (Samsung BKS account), can't proceed
+    if (wallet.activeAccount.hdPath) {
```

Comment:
Do we need to implement something for Samsung BKS users to get an auth token?

Reply:
Yeah, we should. I'm yet to look into that.